spec/support/auth_helpers.rb

```diff
@@ -16,8 +16,8 @@ def stub_env_for_staging_auth(env_field_for_username: :http_user,
   end
 
   def stub_hiring_staff_auth(urn: nil, uid: nil, session_id: 'session_id', email: nil)
-    page.set_rack_session(urn: urn) if urn.present?
-    page.set_rack_session(uid: uid) if uid.present?
+    urn.present? ? page.set_rack_session(urn: urn) : page.set_rack_session(urn: '')
```

Comment:
Maybe this could be
```ruby
if urn.present?
  page.set_rack_session(urn: urn)
elsif uid.present?
  page.set_rack_session(uid: uid)
end
```

Reply:
We need to explicitly set `session[:urn/:uid]` to `''`, to avoid the impossible state of having both present. I will use `if` and `elsif` to eliminate the unnecessarily checking both uid and urn.